pkg/core/native_policy_test.go

```diff
@@ -110,116 +40,15 @@ func TestStoragePrice(t *testing.T) {
 		n := chain.contracts.Policy.GetStoragePriceInternal(chain.dao)
 		require.Equal(t, int64(native.DefaultStoragePrice), n)
 	})
-
-	testGetSet(t, chain, chain.contracts.Policy.Hash, "StoragePrice", native.DefaultStoragePrice, 1, 10000000)
 }
 
 func TestBlockedAccounts(t *testing.T) {
 	chain := newTestChain(t)
-	account := util.Uint160{1, 2, 3}
-	policyHash := chain.contracts.Policy.Metadata().Hash
-
 	transferTokenFromMultisigAccount(t, chain, testchain.CommitteeScriptHash(),
 		chain.contracts.GAS.Hash, 100_00000000)
 
 	t.Run("isBlocked, internal method", func(t *testing.T) {
```

Comment:
These checks can be moved too, I think. We're testing `native` package here anyway.

Reply:
We're testing internal method here and it uses dao to retrieve blocked accounts. Our native tests can't be dependant on dao.